fish_prompt.fish

```diff
@@ -627,6 +628,16 @@ function fish_prompt -d 'bobthefish, a fish theme optimized for awesome'
   set -l __bobthefish_left_black_arrow_glyph  \uE0B2
   set -l __bobthefish_left_arrow_glyph        \uE0B3
 
+  if [ "$theme_powerline_fonts" = "no" ]
+    set __bobthefish_branch_glyph            ''
```

Comment:
we don't use most of the non-arrow glyphs, but we do use branch. is there a good unicode character  that approximates this?